src/airstack/execute_query.py

```diff
@@ -87,16 +90,24 @@ def queries_object(self):
             object: execute popular query obiect
         """
         from airstack.popular_queries import ExecutePopularQueries
```

Comment:
I know this didn't go in this commit. But this too needs to go to the top of the file with other imports.